scripts/deploy.sh

```diff
@@ -68,3 +71,6 @@ fi
 
 # Deploy subgraph
 graph deploy aragon/aragon-court${SUBGRAPH_EXT} --ipfs ${IPFS_NODE} --node ${GRAPH_NODE}
+
+# Remove manifest
+rm subgraph.yaml
```

Comment:
Why would you want to always remove this? Couldn't we make it optional?

Reply:
Initially, it was being backed up to `subgraph.<network>.yml` but then someone dropped all those files. If we are not going to keep them, why backing up only one that won't be super helpful since it will be regenerated every time we deploy?